docker-compose.yml

```diff
@@ -20,11 +20,24 @@ services:
     <<: *netbox
     depends_on:
     - redis
+    - postgres
     entrypoint:
+    - "/sbin/tini"
```

Comment:
I still don't get why the init system is necessary. What is the drawback if the entrypoint is just `/bin/sh` or `/opt/netbox/venv/bin/python3`?